### DISCUSSION TITLE
Customizable UI parts

### DISCUSSION THREAD

**Discussion by sestegra** [coder/code-server#2597] [Ideas] [answered]
After a discussion on Slack about the customization of page title and icons and my personal experience to edit the code to change the login page content, I would like to start a discussion about the ability to customize these UI parts.

What are you current requirements about such customization?

**Comment by sestegra** [comment]
On my side, I would need to update the login page text.
Would it be possible with configuration file?

**Comment by nhooyr** [comment, marked as answer]
It's unlikely we'll add configuration options for such changes. You can modify the source easily. Please give CONTRIBUTING.md a read!
